sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/CustomDigitalTwin.cs

```diff
@@ -31,18 +30,33 @@ internal class CustomDigitalTwin
         public MyCustomComponent Component1 { get; set; }
     }
 
+    internal class MyCustomDigitalTwinMetadata
```

Comment:
I took the time to make the custom digital twin type even better, but this is independent of the core change here.